documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,200 @@
+
+1. From source code
+
+<details open>
+<summary>Commands to start standalone from source code</summary>
+<p>
+
+Checkout the source code:
+```console
+$ git clone https://github.com/pravega/pravega.git
+$ cd pravega
+```
+Build the Pravega standalone mode distribution:
+
+```console
+$ ./gradlew startStandalone
+```
+
+</p>
+</details>
+
+2. From installation package
+
+<details open>
+<summary>Commands to start standalone from installation package</summary>
+<p>
+Download the Pravega release from the [GitHub Releases](https://github.com/pravega/pravega/releases).
+
+```console
+$ tar xfvz pravega-<version>.tgz
+```
+Download and extract either tarball or zip files. Follow the instructions provided for the tar files (same can be applied for zip file) to launch all the components of Pravega on your local machine.
+
+Run Pravega Standalone:
+
+```console
+$ pravega-<version>/bin/pravega-standalone
+```
+
+</p>
+</details>  
+
+
+# 4. Bootstrapping the Project.
+
+The easiest way to bootstrap a sample application against Pravega is to run the following command in a folder of your choice. 
+```console
+$ gradle init --type java-application
+```
+Add the below snippet to dependencies section of build.gradle in the app directory
+```groovy
+// https://mvnrepository.com/artifact/io.pravega/pravega-client
+implementation group: 'io.pravega', name: 'pravega-client', version: '0.8.1'
+```
+Invoke `gradle run` to run the project.
+
+ 
+<details open>
+<summary>Expected output</summary>
+<p>
+
+```console
+$ gradle run
+
+> Task :app:run
+Hello World!
+
+BUILD SUCCESSFUL in 890ms
+2 actionable tasks: 2 executed
+```
+
+</p>
+</details>
+
+# 4.1 Create a Pravega Stream
+
+Let’s first get to know Pravega’s client APIs by creating a stream with a fixed scaling policy of 1 segment. We’ll need a StreamConfiguration to define this:
+```java
+StreamConfiguration streamConfig = StreamConfiguration.builder()
+        .scalingPolicy(ScalingPolicy.fixed(1))
+        .build();
+```
+With this streamConfig, we can create streams that feel a bit more like traditional append-only files. Streams exist within scopes, which provide a namespace for related streams. We use a [StreamManager](https://pravega.io/docs/latest/javadoc/clients/io/pravega/client/admin/StreamManager.html) to tell a Pravega cluster controller to create our scope and our streams. Since we are using a standalone Pravega let's use the below controller address.
+```java
+URI controllerURI = URI.create("tcp://localhost:9090");
+```
+The code to create a Pravega stream is as follows
+```java
+try (StreamManager streamManager = StreamManager.create(controllerURI)) {
+        streamManager.createScope("examples");
+        streamManager.createStream("examples", "helloStream", streamConfig);
+}
+```
+Executing the above lines should ensure we have created a Pravega scope called `examples` and a Pravega Stream called `helloStream`.
+
+# 4.2 Create a Pravega Event Writer and write events into the stream.
```

Comment:
Same about ending dots in titles and title size (is a subsection in section 4).

Reply:
fixed it.